lib/newman_conway.py

```diff
@@ -4,7 +4,31 @@
 # Space Complexity: ?
 def newman_conway(num):
     """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n^2)
+        Space Complexity: O(n)
     """
```

Comment:
👍  Since you are looping through the list of numbers from 1 to n, you have time complexity O(n)